packages/timeline-state-resolver/src/integrations/vmix/index.ts

```diff
@@ -793,6 +794,27 @@ export class VMixDevice extends DeviceWithState<VMixStateExtended, DeviceOptions
 					timelineId: '',
 				})
 			}
+			if (input.listFilePaths !== undefined && !_.isEqual(oldInput.listFilePaths, input.listFilePaths)) {
```

Comment:
`input.listFilePaths !== undefined`
Is this correct?

When deactivating a rundown, the studio will go into the studio baseline state which will most likely not have a timeline object defining this list. So the clips wont be cleared? (Im assuming, I havent read enough to say for certain)

Reply:
Good catch, that was indeed incorrect. I've edited it so that clips will be cleared when returning to a baseline which does not have a timeline object defining this list.